yoti-sdk-api/src/main/java/com/yoti/api/client/identity/NotificationRepresentation.java

```diff
@@ -0,0 +1,109 @@
+package com.yoti.api.client.identity;
+
+import static com.yoti.api.client.spi.remote.util.Validation.notNullOrEmpty;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public final class NotificationRepresentation {
```

Comment:
Maybe name this `NotificationConfig` or `PushNotification` or `ShareSessionPushNotification` or `ShareSessionNotification`


---

Because docs already has "Session" it might be a good idea to name the share session `ShareSession` instead of simply `Session`. And maybe use that prefix on all "share" related stuff.